warehouse/templates/manage/settings.html

```diff
@@ -20,6 +20,11 @@
 {% block main %}
   <h2>{% trans %}Project settings{% endtrans %}</h2>
 
+  {% trans project_upload_limit=project.upload_limit | float | filesizeformat(true) , help_url=request.help_url(_anchor='file-size-limit') %}Project upload limit is:
+  <a href="{{ help_url }}">{{ project_upload_limit }} </a>{% endtrans %}<br><br>
+  {% trans project_total_limit=project.total_size_limit | float | filesizeformat(true) , help_url=request.help_url(_anchor='project-size-limit') %}Project total size limit is:
+  <a href="{{ help_url }}">{{ project_total_limit }} </a>{% endtrans %}<br><br>
+
```

Comment:
Instead of making the value a link, can we add a parenthetical after it, something like:

* Project file size limit: 150 MiB <a href="">(request an increase)</a>
* Project total size limit: 10 GiB <a href="">(request an increase)</a>

Reply:
Yeah sure

Reply:
![Screenshot from 2020-07-07 21-05-12](https://user-images.githubusercontent.com/35963992/86805540-92932b80-c095-11ea-9ffb-12b8c9cb4a1e.png)

Reply:
@di  Committed the changes. All builds are successful. Travis is not working properly. Maybe look into that?